README.md

```diff
@@ -0,0 +1,23 @@
+# NI JavaScript Style Guide
```

Comment:
Can we include TypeScript in the heading too? Even though we won't support linting it from day 1, it's good to express our intent for the repo.

Reply:
You bet!